composeApp/src/commonMain/kotlin/data/db/DBHelper.kt

```diff
@@ -0,0 +1,39 @@
+package data.db
+
+import app.cash.sqldelight.ColumnAdapter
+import app.cash.sqldelight.db.SqlDriver
+import com.monstarlab.kmp.ProductDB
+import com.monstarlab.kmp.ProductDatabase
+import data.mapper.ProductDBMapper
+import domain.model.Product
+
+class DBHelper(sqlDriver: SqlDriver, private val mapper: ProductDBMapper) {
```

Comment:
Not really a fan of the naming here and what it represents. The name suggest some generic database utility class while in reality its a data source for the Products

Think this should a `ProductDatabaseDataSource`  that implements `ProductsDataSource` interface